nomenclature/processor/region.py

```diff
@@ -274,24 +274,25 @@ def from_directory(cls, path: DirectoryPath):
         """
         mapping_dict: Dict[str, RegionAggregationMapping] = {}
         errors: List[ErrorWrapper] = []
-        for file in path.glob("**/*.yaml"):
-            try:
-                mapping = RegionAggregationMapping.from_file(file)
-                if mapping.model not in mapping_dict:
-                    mapping_dict[mapping.model] = mapping
-                else:
-                    errors.append(
-                        ErrorWrapper(
-                            ModelMappingCollisionError(
-                                model=mapping.model,
-                                file1=mapping.file,
-                                file2=mapping_dict[mapping.model].file,
-                            ),
-                            "__root__",
+        for file in path.glob("**/*"):
+            if file.suffix in {".yaml", ".yml"}:
```

Comment:
Suggest to use this more elaborate formulation to avoid double-indenting
```suggestion
        for file in [f for f in path.glob("**/*") if f.suffix in {".yaml", ".yml"}]:
```